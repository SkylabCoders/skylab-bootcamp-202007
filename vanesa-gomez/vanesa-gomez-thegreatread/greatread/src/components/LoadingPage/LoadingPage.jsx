import React from 'react';
import './loadingPage.scss';

function LoadingPage() {
    return (
        <div className="spinner-container">
            <img
                className="spinner"
                alt="spinner"
                width="150px"
                height="150px"
                src="https://static.wixstatic.com/media/332553_2c00ce4bd4e84416bae88ea4d1405950~mv2.gif"
            />
        </div>
    );
}

export default LoadingPage;
