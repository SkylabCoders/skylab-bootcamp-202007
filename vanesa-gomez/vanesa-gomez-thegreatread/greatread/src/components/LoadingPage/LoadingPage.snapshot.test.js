import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import LoadingPage from './LoadingPage';
describe('LoadingPage', () => {
    const loadingPageTree = renderer.create(
        <Router>
            <LoadingPage />
        </Router>
    );
    it('should match', () => {
        expect(loadingPageTree).toMatchSnapshot();
    });
});
