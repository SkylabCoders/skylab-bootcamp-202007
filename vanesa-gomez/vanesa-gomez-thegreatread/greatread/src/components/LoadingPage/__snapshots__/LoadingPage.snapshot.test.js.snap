// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should match 1`] = `
<div
  className="spinner-container"
>
  <img
    alt="spinner"
    className="spinner"
    height="150px"
    src="https://static.wixstatic.com/media/332553_2c00ce4bd4e84416bae88ea4d1405950~mv2.gif"
    width="150px"
  />
</div>
`;
